app/views/publisher_information/publisher_updates.govspeak.md

```diff
@@ -1,9 +1,2 @@
-### Scheduling <span class="govuk-caption-m">22 October 2018</span>
-You can now schedule documents in to publish in the future
-
-
-
-### Inline images <span class="govuk-caption-m">16 September 2018</span>
-Inline images are now available for News stories
-
-
+### Content Publisher is available in beta <span class="govuk-caption-m">29 November 2018</span>
+Beta partners can now access Content Publisher to publish news stories and press releases. Most features are available, [some will be released later](https://content-publisher.integration.publishing.service.gov.uk/beta-capabilities).
```

Comment:
This link currently points to our integration environment, if you change it to be `[some will be released later](/beta-capabilities)` then it should work for all.

Reply:
Done